### PR TITLE
SHM transport is enabled by default [9814]

### PR DESCRIPTION
**Pull request by IkerLuengo**

### CODE REVIEW

File: docs/fastdds/transport/transport.rst [hunk 1]
```diff
@@ -31,6 +32,8 @@ can choose the one that best suits their requirements, or create their own.
   (see :ref:`transport_tcp_tcp`).
 
 - **SHM**: Shared memory communication among entities running on the same host
```
Comment (EduPonz) [inline docs/fastdds/transport/transport.rst:34]:
```suggestion
- **SHM**: Shared memory communication among entities running on the same host.
```

File: docs/fastdds/transport/shared_memory/shared_memory.rst [hunk 2]
```diff
@@ -153,10 +153,12 @@ such as Wireshark.
...
 
-SHM transport is not enabled by default.
-To enable SHM transport in a DomainParticipant, you need to
-create an instance of SharedMemTransportDescriptor and add it to the user transport list of the
-DomainParticipant.
+*Fast DDS* enables a SHM transport by default.
+Nevertheless, the application can enable other SHM transports if needed.
+To enable a new SHM transport in a :ref:`dds_layer_domainParticipant`, first
+create an instance of :ref:`transport_sharedMemory_transportDescriptor`,
+and add it to the user transport list of the :ref:`dds_layer_domainParticipant`.
```
Comment (EduPonz) [inline docs/fastdds/transport/shared_memory/shared_memory.rst:160]: What are the settings of the default one?

**Review by EduPonz — CHANGES_REQUESTED** [review]
You'll need to also update CMake options section and library overview

File: docs/installation/configuration/cmake_options.rst [hunk 3]
```diff
@@ -77,12 +76,11 @@ dependency on other options.
         - ``ON`` ``OFF``
         - ``OFF``
     *   - :class:`SHM_TRANSPORT_DEFAULT`
-        - Adds Shared Memory transport (SHM) to the default transports. It is set to ``ON`` if |br|
-          :class:`EPROSIMA_BUILD` is ``ON``. Please refer to :ref:`SHM <transport_sharedMemory_sharedMemory>` section
-          for more information |br|
+        - Adds Shared Memory transport (SHM) to the default transports.
+          Please refer to :ref:`SHM <transport_sharedMemory_sharedMemory>` section for more information |br|
           on *Fast DDS* SHM transport.
```
Comment (EduPonz) [inline docs/installation/configuration/cmake_options.rst:81]: I had to scroll sideways to see the full table
```suggestion
        - Adds Shared Memory transport (SHM) to the default transports.
          Please refer to :ref:`SHM <transport_sharedMemory_sharedMemory>` section |br|
          for more information on *Fast DDS* SHM transport.
```

**Review by EduPonz — CHANGES_REQUESTED** [review]

**Review by EduPonz — APPROVED** [review]

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/650/)

**Pull request merged**